homeassistant/components/bond/light.py

```diff
@@ -113,7 +113,7 @@ async def async_setup_entry(
     )
 
     platform.async_register_entity_service(
-        SERVICE_SET_LIGHT_POWER_BELIEF,
+        SERVICE_SET_LIGHT_POWER_TRACKED_STATE,
         {vol.Required(ATTR_POWER_STATE): vol.All(cv.boolean)},
         "async_set_power_belief",
```

Comment:
I wonder if we should change this as well.

Reply:
Actually since the api itself uses this name, and its only exposed to developers, I think we should leave it.